### DISCUSSION TITLE
Copilot is not working in Student Benifts

### DISCUSSION THREAD

**Discussion by JoaquinArruiz** [community/community#60914] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Hi! My copilot is not activated on my [account settings](https://github.com/settings/copilot). I already got accepted at github education as a student 2 days ago but the benefits dont show up, I couldn't find what is average time that I have to wait for the benefits to be avaible.

**Comment by the-officialjosh** [comment, marked as answer]
It will be rolled out to you soon wait, you should get the PRO badge first most times

**Reply by JoaquinArruiz** [reply]
It rolled out! Just needed to wait 4 to 5 days. Thank you!

**Reply by Henok-Enyew** [reply]
> It will be rolled out to you soon wait, you should get the PRO badge first most times

hey i have a pro badge and i've been using copilot code completions but it says upgrade to pro to use like since yesterday what is the probelm, it also says this restrictions will be released on march 
31

**Comment by zkllong-21** [comment]
I have the same issue as well

**Comment by zkllong-21** [comment]
yes, thank you
&nbsp;


1831868128
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Ignacio ***@***.***&gt;; 
发送时间: 2023年7月28日(星期五) 凌晨1:04
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [community/community] Copilot is not working in Student Benifts (Discussion #60914)





 
Hi. Did you solve your issue?
